### ISSUE TITLE
spotifgopher.go:166: Could not get info: trackid

### ISSUE THREAD

**Issue by philjones88** [wtfutil/wtf#760]
## What's broken?
Can't open on MacOS suddenly. It worked yesterday.

Shows a broken layout with the message:

```
NAMES2019/11/21 10:30:50 spotigopher.go:166: Could not get info: trackid
```

Using:
MacOS 10.14.6 (18G1012)
iTerm2 3.3.7
Wtfutil: lastest on brew (`--version` shows `dev`)
go version go1.12.9 darwin/amd64

**Comment by philjones88** [comment]
Realised this is the spotify stuff. I went to Spotify and started playing a track. Then re-opened iTerm2 + wtfutil and it works again.

**Comment by senorprogrammer** [comment]
Thanks for reporting this.

**Comment by senorprogrammer** [comment]
Opened a PR on `sticreations/spotigopher` for this one: https://github.com/sticreations/spotigopher/pull/4

**Comment by senorprogrammer** [comment]
Given the lack of action in the `sticreations/spotigopher` I'm going to fork that into here and fix it locally.

**Comment by senorprogrammer** [comment]
This fix will be included as part of release 0.26.0. If you're building from source, it's in `master` now.

**Comment by sticreations** [comment]
> Given the lack of action in the `sticreations/spotigopher` I'm going to fork that into here and fix it locally.

Sorry. Had quite the Off-Time because of personal reasons. MR is now accepted.
